solara/components/file_input.py

```diff
@@ -0,0 +1,103 @@
+"""
+FileInput component.
+"""
+
+
+from typing import Callable, List, Optional, Union, cast
+
+from ipyvuetify.extra import FileInput as ExtraFileInput
+
+import solara
+import solara.hooks as hooks
+from solara.components.file_drop import FileInfo
+
+
+@solara.component
+def FileInput(
```

Comment:
```suggestion
def InputFile(
```
We have postfixed all input components with Input, such as InputText/Int/Float, so I think it makes sense to rename it like this.